src/_pyinstaller_hooks_contrib/hooks/stdhooks/hook-parso.py

```diff
@@ -0,0 +1,14 @@
+#-----------------------------------------------------------------------------
```

Comment:
This is what the linter is moaning about.
```suggestion
# ----------------------------------------------------------------------------
```

Reply:
Likewise at the end of the copyright header...

Reply:
Fixed